submissions/SamVal007/task-js-dom/dom.js

```diff
@@ -0,0 +1,148 @@
+    born: "11 February 1982 (age 38) Melbourne, Victoria, Australia",
+    CurrentRanking: "2 (as of 21 December 2020)",
+    CareerWinnings: "£5,318,112",
+    pic: "./img/Neil Robertson.jpg",
+    descr: [
+      "Neil Robertson (born 11 February 1982) is an Australian professional snooker player. He made his first breakthrough into the top professional ranks in the 2006–07 season. He won the 2010 World Championship and was the world number one later in the same year, a ranking that he attained again in 2013 and 2014. On 10 November, Neil Robertson won the 2019 Champion of Champions tournament for the second time in his career and secured his 19th ranking event.",
+      "Robertson is the only Australian to have won a ranking event, and was undefeated in his first six televised finals. Robertson is also one of thirteen players to win both the world and UK titles, and one of eleven to win the Triple Crown of World Championship, UK Championship and Masters. As a prolific break-builder, Robertson has compiled more than 750 century breaks in professional competition. During the 2013–14 season he became the first player to make 100 centuries in a single season.",
+      "Robertson is the most successful player from outside the United Kingdom in the sport's history. He plays left-handed.",
+      "In June 2016, he became ambassador of electronic snooker simulator app Snooker Live Pro. He was an avid gamer but gave up the hobby in April 2017, believing he was spending too much time playing games and it was affecting his snooker form."
+    ]
+  },
+]
+
+const playerPhoto = document.querySelector('.photo');
+const playerName = document.querySelector('.player-name');
+const playerborn = document.querySelector('.born');
+const playerCurrentRanking = document.querySelector('.CurrentRanking');
+const playerCareerWinnings = document.querySelector('.CareerWinnings');
+const playerDescription = document.querySelector('.descr');
+const NAV = document.querySelector('nav');
+
+let currentCarIndex = "0";
+
+const toggleActiveLink = (newActiveEl) => {
+  document.querySelector('.active').classList.toggle('active');
+  newActiveEl.classList.toggle('active');
+}
+
+const generateCarDescription = (id) => {
+  const fragment = document.createDocumentFragment();
+  const div = document.createElement('div');
+
+  PLAYERS[id].descr.forEach(item => {
+    const p = document.createElement('p');
+
+    p.textContent = item;
+    div.appendChild(p);
+  });
+
+
+  fragment.appendChild(div);
+  return fragment.firstElementChild.innerHTML;
+}
+
+const displayContent = (activeElem) => {
+  playerPhoto.setAttribute('src', PLAYERS[activeElem.id].pic);
+  playerName.textContent = PLAYERS[activeElem.id].name;
+  playerborn.textContent = PLAYERS[activeElem.id].born;
+  playerCurrentRanking.textContent = PLAYERS[activeElem.id].CurrentRanking;
+  playerCareerWinnings.textContent = PLAYERS[activeElem.id].CareerWinnings;
+}
+
+const handler = ({
+  target
+}) => {
```

Comment:
```suggestion
const handler = ({ target  }) => {
```
Maybe it will be more readable for you.